src/games/static_fitness.c

```diff
@@ -396,7 +396,7 @@ int igraph_static_power_law_game(igraph_t *graph,
 
     if (exponent_in >= 0) {
         if (exponent_in < 2) {
-            IGRAPH_ERROR("In-degree exponent must be >= 2.", IGRAPH_EINVAL);
+            IGRAPH_ERRORF("In-degree exponent must be either negative or >= 2, got %g.", IGRAPH_EINVAL, exponent_in);
```

Comment:
I would not mention negative values here.

Different high-level interfaces have different ways to indicate that the graph should be undirected. For some of them, talking about a negative exponent will be confusing to users. A negative exponent is just an idiosyncratic way in the C interface to indicate that the graph must be undirected.